contents/linux/shell.md

```diff
@@ -0,0 +1,180 @@
+<frontmatter>
+  title: Introduction to Linux Shell
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Linux bash shell 
+
+Author: [Wang Junming](https://github.com/junming403)
+
+## Overview
+
+This chapter assumes that the reader is familiar with general concepts of operating systems. If you are not familiar with basic operating system concepts, I recommend you pick up the basic concepts first. The classic textbook of operating system **Mordern Operating Systems by Tanenbaum** is highly recommended. 
```

Comment:
Do we really need that much familiarity with Operating Systems to understand how to use the basic bash shell?

The textbook is used for introductory courses such as Introduction to Operating Systems. Seems a bit much here to read that before coming here?

Why not you just link a simple article or something?

Reply:
While in my understanding, one can only make good use of bash shell only if he understand how  OS kernel works. Indeed I agree that without learning basic concepts of OS, one can still use OS, but it's always good to understand how tools works before using the tool? I will mention this Book is a not prerequisite to read this article instead.